libnmstate/nm/active_connection.py

```diff
@@ -336,10 +336,9 @@ def _activation_progress_check(self):
             )
 
     def _fallback_checker_callback(self, _user_data):
-        self._nm_dev = get_nm_dev(
-            self._ctx, self._iface_name, self._iface_type
-        )
-        if self._nm_dev:
+        nm_dev = get_nm_dev(self._ctx, self._iface_name, self._iface_type)
+        if nm_dev:
+            self._nm_dev = nm_dev
```

Comment:
I think this change is not related with this commit.

Reply:
Moved to https://github.com/nmstate/nmstate/pull/1486

Reply:
Thank you!